app/controllers/additional_details/submissions_controller.rb

```diff
@@ -44,6 +44,8 @@ def create
     @submissions = @protocol.submissions
     @line_item = LineItem.find(submission_params[:line_item_id])
     @service_request = @line_item.service_request
+    @permission_to_edit = @protocol.project_roles.where(identity: current_user, project_rights: ['approve', 'request']).any?
```

Comment:
Gotchya. Well let's change all those over to the Identity method in your upcoming refactor.